aunt-leahs-app/src/components/textField/textField.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { TextField, Typography, makeStyles } from '@material-ui/core';
+import '../../styles.css';
+
+const useStyles = makeStyles({
+    root:{
+        border: '2px solid #00A19B',
+        boxSizing: 'border-box',
+        fontFamily: 'Tahoma',
+        fontStyle: 'normal',
+        fontWeight: 'normal',
+        fontSize: '18px',
+        lineHeight: '22px',
+        padding: '0px 5px',
+    },
+});
```

Comment:
Can you move this to styles.css ?

Reply:
This calls 'makeStyles' which is a function I imported from material-ui: " import { TextField, Typography, makeStyles } from '@material-ui/core' ". So I'm not sure if it will function the same if it is moved to styles.css

Reply:
You should be able to import makeStyles to styles.css. Give it a try and let me know if it works.